3rdPartyCypress/cypress/integration/worldServiceCoUkRedirect.js

```diff
@@ -0,0 +1,36 @@
+import services from '../../../src/app/lib/config/services';
+
+Object.keys(services).forEach(service => {
+  const notWSServices = ['news', 'cymrufyw', 'naidheachdan', 'default']; // Not WS
+
+  if (notWSServices.includes(service)) {
+    return;
+  }
+  describe('WS Redirects', () => {
+    it(`should redirect *bbc.com/${service}`, () => {
+      const urlsTotest = [
+        `https://www.bbc.co.uk/${service}`,
+        `https://www.bbc.co.uk/${service}/articles/a0000000000o`,
+        `https://www.bbc.co.uk/${service}/articles/a0000000000o.amp`,
+      ];
+
+      urlsTotest.forEach(urlToTest => {
+        const slashLoc = urlToTest.indexOf('/', 8);
+        cy.request({
+          url: urlToTest,
+          followRedirect: false,
+        }).then(resp => {
+          expect(resp.status).to.eq(301);
```

Comment:
For services arabic, igbo, persian, pidgin, yoruba, the status code is currently a 302, not a 301.

Reply:
Yes and that will be corrected before this is merged

Reply:
https://github.com/bbc/simorgh/issues/2814